buildSrc/src/main/groovy/com/uber/okbuck/OkBuckGradlePlugin.groovy

```diff
@@ -37,6 +37,17 @@ import org.gradle.api.artifacts.repositories.FlatDirectoryArtifactRepository
 import org.gradle.api.artifacts.repositories.IvyArtifactRepository
 import org.gradle.api.artifacts.repositories.MavenArtifactRepository
 
+// Dependency Tree
+//
+//                okbuck
+//          /                \
+//         /             okbuckClean
+//        /        /          |          \
+//       /     :p1:okbuck   :p2:okbuck   :p3:okbuck     ...
+//      |        /           /               /
+//      setupOkbuck
```

Comment:
This does not look right `setupOkbuck` should run before all other `okbuck` tasks